dqc_us_rules/dqc_us_0004.py

```diff
@@ -36,13 +64,19 @@ def _assets_eq_liability_equity(modelXbrl):
         for fact_group in fact_groups:
             fact_assets = fact_group[_ASSETS_CONCEPT]
             fact_liabilities = fact_group[_LIABILITIES_CONCEPT]
-            if fact_assets.context is not None and fact_assets.context.instantDatetime is not None:
+            if ((fact_assets.context is not None and
+                 fact_assets.context.instantDatetime is not None)):
+
                 dec_assets = inferredDecimals(fact_assets)
                 dec_liabilities = inferredDecimals(fact_liabilities)
                 min_dec = min(dec_assets, dec_liabilities)
-                if _min_dec_valid(min_dec) and _values_unequal(fact_assets.xValue, fact_liabilities.xValue, min_dec):
+                if ((_min_dec_valid(min_dec) and
+                     _values_unequal(
+                         fact_assets.xValue, fact_liabilities.xValue, min_dec
+                     ))):
```

Comment:
I don't really like the triple closing parens here, but given the odd layout, I'm not sure there's a more correct alternative. I'd suggest that this is an instance where going over the 80 character limit is preferable, but I'm not sure how strict we're trying to be for this public facing repo.